scripts/utils-wolfssl.sh

```diff
@@ -70,7 +71,7 @@ install_wolfssl() {
         printf "Done.\n"
 
         printf "\tBuild wolfSSL ${WOLFSSL_TAG} ... "
-        make -j$NUMCPU &>> $LOG_FILE
+        make -j$NUMCPU 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/utils-wolfssl.sh line 74:
        make -j$NUMCPU 2>&1 >> $LOG_FILE
               ^-----^ SC2086 (info): Double quote to prevent globbing and word splitting.
                       ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                               ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```